### PR TITLE
Re-synced wired test cases with params as mandated by eval-test05

### PR DESCRIPTION
**Pull request by sergio-mena**

### CODE REVIEW

File: test/rmcat-wired-test-suite.cc [hunk 1]
```diff
@@ -173,12 +182,13 @@ RmcatTestSuite::RmcatTestSuite ()
...
-    tstartTC55.push_back (2);  tstopTC55.push_back (299);
-    tstartTC55.push_back (3);  tstopTC55.push_back (299);
-    tstartTC55.push_back (4);  tstopTC55.push_back (299);
+    tstartTC55.push_back (10);  tstopTC55.push_back (299);
+    tstartTC55.push_back (20);  tstopTC55.push_back (299);
+    tstartTC55.push_back (30);  tstopTC55.push_back (299);
+    tstartTC55.push_back (40);  tstopTC55.push_back (299);
     RmcatWiredTestCase * tc55 = new RmcatWiredTestCase{bw, 10, qdel, "rmcat-test-case-5.5-fixfps"};
-    tc55->SetSimTime (300);  // simulation time: 300s
+    tc55->SetCapacity (3.5 * (1u << 20));  // bottleneck capacity: 3.5 Mbps
```
Comment (zhuxqing) [inline test/rmcat-wired-test-suite.cc:190]: According to the draft, there should be no other background traffic in this experiment. So the Capacity should be set at the default of 4Mbps.

**Review by zhuxqing — COMMENTED** [review]
Overall the changes look good. I just have one questions regarding the BW choice for Test Case 5.5 (round trip time fairness).  See my note for the conversation.  

BTW, it occurs to me that it's pretty painful to go through the code and change to manually change test settings each time. We may want to add a TODO at the beginning on "load test cases from json files" for future improvement. This, of course, is a "nice to have" feature :)

File: test/rmcat-wired-test-suite.cc (hunk 1 above)
Comment (sergio-mena) [reply test/rmcat-wired-test-suite.cc:190]: TC 5.5 states that "the topology is the same as in Section 5.4". The Path characteristics of Section 5.5 don't indicate any reference bottleneck capacity, whereas Section 5.4 indicates a reference bottleneck capacity of 3.5Mbps (and ratio of 1.0).
I propose we look at it together in our next meeting.

**Comment by sergio-mena** [comment]
Thanks for merging.
I totally agree on the json data to set up the tests. I have added a "TODO (deferred)" at the beginning of test case setup in the code.
(Will wait for the final status of TC5.5 before creating another pull request)

**Pull request merged**